### ISSUE TITLE
No new Effects

### ISSUE THREAD

**Issue by compoundbow83** [wez/govee2mqtt#195]
### Govee Device SKU

H70A1

### Govee2MQTT Version

2024.04.29-30cf7732

### Describe the issue

For some time now I have also seen the possibility of selecting an effect with the H70A1 strips.

The problem is that I don't see any new effects in the Home Assistant that I create in the Govee app.

I only see standard effects and a few older effects that I have created myself.

But what I notice with the H169A strips is that I only see older effects that I have created in the Govee app. I don't see any newly created effects there either.

Am I doing something wrong here, or is there a problem with the API?

### Startup Diagnostics

I have already restarted the addon and Home Assistant several times

**Comment by OmenBoy** [comment]
yup I'm having the same issue, only old scene's show up, nothing new.

**Comment by compoundbow83** [comment]
I even wrote to Govee Support because I think it's due to the API. But there were only useless answers

**Comment by OmenBoy** [comment]
This makes me want to switch everything to WLED. I’ve read somewhere that Govee LEDs are compatible with some controllers. I mainly went with Govee for the Dreamview stuff, but I really don’t use that feature as much as I thought I would.

**Comment by Hubbsjp** [comment]
I have seen some recently created effects come into HA, but not others.  No rhyme or reason.  Restarted the addon and ha like others.  Very frustrating.  wish there was a solution to this, or a reason why I can't get the effect.

**Comment by mexigabacho** [comment]
I believe I have this same issue.  But, in my situation, the new DIY effects appear for one Govee device, but not another.  

I wonder if this is a Govee API/device issue or not, but I've tried just about everything I could think of (restarting various pieces, clearing cache, etc..).   I can add a DIY effect, and after a while, it will show up in one device (H7060 - FloodLights), but not on the one I actually created it on in the Govee app (H705D - Permanent hanging lights).

As an aside, the Govee app is the most convoluted thing I think I've even seen created by a smarthome/lighting company, but I did check and re-check everything I could think of.   I plan on spending some time trying some things on the CLI with curl/etc.. to try and confirm if this is actually a Govee API issue or something else.  I'll report back after I figure out how to do that.

**Comment by jarivanb** [comment]
So does anyone have a solution for this? Or is it still not working?
